api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -124,8 +125,25 @@ message DownstreamTlsContext {
   OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
 }
 
+// TLS key log configuration.
+// The key log file format is "format used by NSS for its SSLKEYLOGFILE debugging output" (text taken from openssl man page)
+message TlsKeyLog {
+  // The path to save the TLS key log.
+  string tls_keylog_path = 1 [(validate.rules).string = {min_len: 1}];
+
+  // At least one of src or dst must be specified, or the config will be rejected as invalid.
+
+  // The src IP address that will be used to filter the connection which should save the TLS key log
+  // If it is not set, any src IP address  will be matched.
+  repeated config.core.v3.CidrRange tls_keylog_src = 2;
```

Comment:
Should this be specified as local and remote address, instead of source and dest?

Reply:
fixed